crypto/arm_arch.h

```diff
@@ -126,4 +126,60 @@ extern unsigned int OPENSSL_armv8_rsa_neonized;
 
 # define MIDR_IS_CPU_MODEL(midr, imp, partnum) \
            (((midr) & MIDR_CPU_MODEL_MASK) == MIDR_CPU_MODEL(imp, partnum))
+
+#if defined(__ASSEMBLER__)
+
+   // Support macros for
```

Comment:
Could you please convert the comments to classic C-style in this file?

Reply:
Done